ee/ucp/admin/install/install-on-azure.md

```diff
@@ -13,54 +13,48 @@ Calico / Azure integration.
+addresses to Kubernetes pods.  The Azure IPAM module requires each Azure virtual
+machine that's part of the Kubernetes cluster to be configured with a pool of IP
+addresses.
 
 There are two options for provisoning IPs for the Kubernetes cluster on Azure
-- Docker UCP provides an automated mechanism to configure and maintain IP pools 
-  for standalone Azure virtual machines. This service runs within the calico-node daemonset 
-  and by default will provision 128 IP address for each node. This value can be 
-  configured through the `azure_ip_count`in the UCP 
-  [configuration file](../configure/ucp-configuration-file) before or after the 
-  UCP installation. Note that if this value is reduced post-installation, existing 
-  virtual machines will not be reconciled, and you will have to manually edit the IP count
-  in Azure. 
-- Manually provision additional IP address for each Azure virtual machine. This could be done
-  as part of an Azure Virtual Machine Scale Set through an ARM template. You can find an example [here](#set-up-ip-configurations-on-an-azure-virtual-machine-scale-set). 
-  Note that the `azure_ip_count` value in the UCP 
-  [configuration file](../configure/ucp-configuration-file) will need to be set
-  to 0, otherwise UCP's IP Allocator service will provision the IP Address on top of 
-  those you have already provisioned.
+- Docker UCP provides an automated mechanism to configure and maintain IP pools
+  for standalone Azure virtual machines. This service runs within the
+  calico-node daemonset and by default will provision 128 IP address for each
+  node. For information on customising this value see [adjusting the ip count value](#adjusting-the-ip-count-value) for more information.
+- Manually provision additional IP address for each Azure virtual machine. This
+  could be done in the Azure Portal, via the Azure CLI `$ az network nic
+  ip-config create` or through an ARM template. You can find an example of an
+  ARM template
+  [here](#manually-provision-ip-address-as-part-of-an-azure-virtual-machine-scale-set).
 
 ## Azure Prerequisites 
 
-You must meet these infrastructure prerequisites in order 
-to successfully deploy Docker UCP on Azure
+You must meet the following infrastructure prerequisites in order 
+to successfully deploy Docker UCP on Azure:
 
-- All UCP Nodes (Managers and Workers) need to be deployed into the same 
-Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
-components could be deployed in a second Azure Resource Group.
-- The Azure Vnet and Subnet must be appropriately sized for your
-  environment, and addresses from this pool are consumed by Kubernetes Pods. For more information, see
-  [Considerations for IPAM
+- All UCP Nodes (Managers and Workers) need to be deployed into the same Azure
+  Resource Group. The Azure Networking components (Virtual Network, Subnets,
+  Security Groups)  could be deployed in a second Azure Resource Group.
```

Comment:
In a second Azure Resource Group but in the same Azure Subscription, correct? Unaware if we support cross-subscription RGs but suspect now (have had customers with 101 Azure Subs and cross-sub anything is painful)

Reply:
Yes, the second RG is in the same subscription. I don't think there is a field to support a second subscription in the azure.json.